docs/core/testing/unit-testing-mstest-migration-from-v1-to-v3.md

```diff
@@ -0,0 +1,290 @@
+---
+title: MSTest migration to v3
+description: Migration guide.
+author: engyebrahim
+ms.author: enjieid
+ms.date: 11/06/2024
+---
+
+# MSTest v3 migration guide
+
+## Overview
+
+This guide assists users in upgrading their MSTest projects from MSTest v1 to MSTest v3. MSTest v3 introduces significant new features, optimizations, and some breaking changes to improve test reliability, execution speed, and compatibility with modern .NET frameworks.
+
+---
+
+### Who Is Impacted?
+
+This guide is intended for projects currently using MSTest v1 via either:
```

Comment:
99% percent of the projects will be .NET Framework, .NET projects don't do direct dll references (at least there is no menu for it in VS), and mstest 1.4.0 is targetting .net standard 1.0, which is technically compatible with .NET, but I doubt anyone uses that. 

so it might be worth pointing out that this is impacting .NET Framework projects   .